module/chunks/chunkVerifier.go

```diff
@@ -18,7 +19,7 @@ import (
 )
 
 type VirtualMachine interface {
-	Run(fvm.Context, fvm.Procedure, state.View, *fvm.Programs) error
+	Run(fvm.Context, fvm.Procedure, state.View, *programs.Programs) error
```

Comment:
Maybe we should define an interface for Programs for flexibility of injecting no-op or other types programs if needed (e.g. testing edge cases, verification, ...)